code/mob/living/critter/mimic.dm

```diff
@@ -112,13 +112,14 @@
 			pounce.handleCast(target)
 			return TRUE
 
+	valid_target(mob/living/C) //apparently they don't have the check for isdead, so gotta do this
```

Comment:
since dead is a subset of incapacitated you could just check `is_incapacitated` and then return parent